### DISCUSSION TITLE
Plotman Analyze Clarification

### DISCUSSION THREAD

**Discussion by XerrorYT** [ericaltendorf/plotman#259] [Q&A]
Looking for some clarification on what the sigma value denotes in the analyze output.
Can anyone shed some light?

**Comment by ericaltendorf** [comment]
Standard deviation.  https://github.com/ericaltendorf/plotman/blob/main/src/plotman/analyzer.py#L124

**Reply by XerrorYT** [reply]
Of course. 
Thankyou Eric.
